resources/styles/global/navbar.less

```diff
@@ -69,14 +70,76 @@
       padding-right: 1rem;
       border-left: 1px solid @tutor-neutral-light;
     }
-    border-right: 1px solid @tutor-neutral-light;
   }
   .navbar-brand {
     font-size: 1.6rem;
     line-height: 2.7rem;
     margin-top: 1px;
   }
 
+  @media screen and (max-width: (@screen-md - 1px)) {
+    .navbar-collapse {
+      max-width: 230px;
+      max-height: 350px;
+      float: right;
+      .tutor-shadow(1);
+      border-top: none;
+    }
+
+    .navbar-nav .navbar-brand {
+      display: none;
+    }
+
+    .navbar-nav > li > a,
+    .navbar-nav > li.book-links > a,
+    .navbar-nav > li.dropdown > ul.dropdown-menu > li > a {
+      height: initial;
+      display: block;
+      padding: 3px 20px;
+      border: none;
+      clear: both;
+      float: none;
+      font-weight: normal;
+      font-size: 14px;
+      line-height: 1.42857143;
+      color: @tutor-neutral;
+      white-space: nowrap;
+
+      &:hover {
+        background-color: @tutor-neutral-bright;
+      }
+    }
+
+    .book-links {
+      border: none;
+    }
+
+    .dropdown {
+      > .dropdown-toggle {
+        display: none;
+      }
+
+      > .dropdown-menu {
+        position: relative;
+        float: none;
+        width: auto;
+        margin-top: -7.5px;
+        padding-top: 0;
+        background-color: transparent;
+        box-shadow: none;
+        display: block;
+      }
+
+      &.is-concept-coach {
+        > .dropdown-menu {
+          border-top: 1px solid @tutor-neutral-light;
+          padding-top: 8px;
+          margin-top: 0;
+        }
+      }
```

Comment:
If it's concept coach, then the list above it is not empty, which means we need to put spacing in between.  There's no way in css of finding out if the child of the child of the element's sibling has specific contents or not, so I had to do it this way.  It can also be used for more useful things, but this was just a hack.  I can't even use the `:empty` selector in the section above it because it's not that it has no children but that it doesn't include children that aren't the `.navbar-brand`.  If someone has a selector for that, I'm all ears.